crates/precompile-currency-swap/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "precompile-currency-swap"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+precompile-utils = { path = "../precompile-utils", default-features = false }
+primitives-currency-swap = { path = "../primitives-currency-swap", default-features = false }
+
+codec = { package = "parity-scale-codec", version = "3.2.2", default-features = false, features = ["derive"] }
+fp-evm = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38", default-features = false }
+frame-support = { default-features = false, git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+frame-system = { default-features = false, git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+getrandom = { version = "0.2", features = ["js"] }
+num_enum = { version = "0.6.0", default-features = false }
```

Comment:
Do we use these dependencies somewhere?

Reply:
We might not, I'll have to check the list as I used our other precompile as a template

Reply:
Reduced the set of the dependencies manually to the amount that I found works (might be able to reduce further via reexports that I missed, but I figured it's good enough).

The `num_enum` is used implicitly by the precompile utils macro that generates selectors from the enum. `getrandom` was not used.